### PR TITLE
pai expansion slots proposal

### PR DESCRIPTION
**Pull request by deltanedas**
read all about it

### CODE REVIEW

**Review by AJCM-git — APPROVED** [review]
This is conceptually fine, the PAIs don't have the full "portable" benefits since there is the barrier of communication with the user. Also the doc implies it, but just to confirm, you can only install a single board, correct?

It would be better (as in more thematically fitting) if it used PDA cartridges instead, but that would add some complexity (adding a whitelist so only PAI can use the program and make the programs give actions, handle edge cases...), so i agree the cards are the most ergonomic option for now.

I will say this is the kind of thing that could be reverted after testing on servers tho

**Comment by deltanedas** [comment]
> you can only install a single board, correct?

yes

> It would be better (as in more thematically fitting) if it used PDA cartridges instead

if it used pda cartridges pros would just put them in their pdas instead of giving them to poor pai

**Comment by AJCM-git** [comment]
> if it used pda cartridges pros would just put them in their pdas instead of giving them to poor pai

read the rest of the paragraph smh

Also how do you plan on doing the board inserting? The best way would prob be using a screwdriver on the PAI to disarm it and then insert the board (would be fun if you also had to glue it since it's unremovable but that may be counter-intuitive)

**Comment by deltanedas** [comment]
> Also how do you plan on doing the board inserting? The best way would prob be using a screwdriver on the PAI to disarm it and then insert the board (would be fun if you also had to glue it since it's unremovable but that may be counter-intuitive)

yes thats how its done, minus the glue. just open the panel then insert and its locked

File: src/SUMMARY.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-Space Wizards Development Wiki
```
Comment (AJCM-git) [inline src/SUMMARY.md:1]: Uhhhhhh

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: src/SUMMARY.md (hunk 1 above)
Comment (deltanedas) [reply src/SUMMARY.md:1]: sus

File: src/SUMMARY.md [hunk 2]
```diff
@@ -142,6 +142,7 @@ Space Station 14
 	- [Player Interaction](en/space-station-14/areas/core/player-interaction.md)
 		- [PR Guidelines](en/space-station-14/areas/core/player-interaction/guidelines.md)
 			- [PDA Messaging](en/space-station-14/areas/core/player-interaction/proposals/pda-messaging.md)
+            - [PAI Expansion Slots](en/space-station-14/areas/core/player-interaction/proposals/pai-expansion-slots.md)
```
Comment (AJCM-git) [inline src/SUMMARY.md:145]:
```suggestion
		- [PAI Expansion Slots](en/space-station-14/areas/core/player-interaction/proposals/pai-expansion-slots.md)
```

**Comment by deltanedas** [comment]
some tab trolling going on

File: src/SUMMARY.md [hunk 3]
```diff
@@ -142,6 +142,7 @@ Space Station 14
 	- [Player Interaction](en/space-station-14/areas/core/player-interaction.md)
 		- [PR Guidelines](en/space-station-14/areas/core/player-interaction/guidelines.md)
 			- [PDA Messaging](en/space-station-14/areas/core/player-interaction/proposals/pda-messaging.md)
+  	- [PAI Expansion Slots](en/space-station-14/areas/core/player-interaction/proposals/pai-expansion-slots.md)
```
Comment (AJCM-git) [inline src/SUMMARY.md:145]: uhh isnt the spacing sitll wrong? it should be under player interaction not at the same level

Comment (deltanedas) [reply src/SUMMARY.md:145]: idk it looks bad i think theres tabs mixed or phone github evil

Comment (AJCM-git) [reply src/SUMMARY.md:145]: yeah try to fix it when you come to your pc or something

Comment (deltanedas) [reply src/SUMMARY.md]: its all good now i think

**Review by AJCM-git — CHANGES_REQUESTED** [review]
fix conflicts and only add yours to the summary without formatting anything else (missed github notif on this one but i promise to handle it quickly next review request)

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**